conf.d/__async_prompt.fish

```diff
@@ -53,7 +53,9 @@ function __async_prompt_spawn -a cmd
     end | read -lz vars
     echo $vars | env $envs fish -c '
     function __async_prompt_signal
-        kill -s "'(__async_prompt_config_internal_signal)'" '$fish_pid'
+        if ps -p '$fish_pid' > /dev/null
```

Comment:
this change does not completely fixes the issue, there is a tiny chance `$fish_pid` got terminated between `ps` and `kill`. Another option would be to suppress `kill`'s error output by

```sh
kill -s "'(__async_prompt_config_internal_signal)'" '$fish_pid' 2>/dev/null
```

Would that be better? hesitating..

Reply:
Thank you for digging the issue and contribution! I prefer suppressing the message than `if ps` approach so I'll update that way and merge.